AspNetCore.SassCompiler/build/AspNetCore.SassCompiler.targets

```diff
@@ -1,5 +1,18 @@
 ﻿<Project>
-  
+
+  <PropertyGroup>
+    <IsOuterBuild Condition="'$(IsOuterBuild)' == ''">false</IsOuterBuild>
+
+    <CompileSassBeforeTargets>Build;ResolveScopedCssInputs;BundleMinify;ResolveProjectStaticWebAssets</CompileSassBeforeTargets>
+    <!-- Use DispatchToInnerBuilds if it is a multi-targeted build -->
+    <CompileSassBeforeTargets Condition="$(IsOuterBuild)">DispatchToInnerBuilds</CompileSassBeforeTargets>
```

Comment:
Do you know what the order is of the "DispatchToInnerBuilds" target compared to the "ResolveScopedCssInputs", "BundleMinify" and "ResolveProjectStaticWebAssets" targets?

If those targets would also only run once and not per target framework we would still need to include them in the `BeforeTargets` to ensure those targets will correctly pick up the generated css files.

Reply:
I believe `DispatchToInnerBuilds` will run before the InnerBuild hence it will run before all those targets.

Reply:
They run for each build which is kind of the problem we are trying to address

Reply:
Okay, if those run for each target framework than it's not a problem